widgets/src/lib/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+	import IconInfo from "../../../Icons/IconInfo.svelte";
+	import IconLightning from "../../../Icons/IconLightning.svelte";
+	import ModelPipelineTag from "../../../ModelPipelineTag/ModelPipelineTag.svelte";
+
+	export let noTitle = false;
+	export let pipeline: string | undefined;
+</script>
+
+<div class="font-semibold flex items-center mb-2">
+	{#if !noTitle}
+		<div class="text-lg flex items-center">
+			<IconLightning classNames="-ml-1 mr-1 text-yellow-500" />
+			Hosted inference API
+		</div>
+		<a target="_blank" href="/docs">
```

Comment:
Clicking on the link button shows `Error. Probably non existent model. Error. Probably non existent model.`

Reply:
yep, limitation of our demo app system here (not the full hf.co env)

Reply:
Even the `Error. Probably non existent model.` in double? (see line 59 of `widgets/src/routes/[...model].svelte`)

Reply:
@Pierrci yes no sure, I had issues locally for some reasons. Maybe a transient SvelteKit bug?